draft-hodges-webauthn-registries.xml

```diff
@@ -42,164 +42,207 @@
         <email>Jeff.Hodges@PayPal.com</email>
       </address>
     </author>
-    
-    <date month="June" year="2016" />
-    
+    <author fullname="Giridhar Mandyam" initials="G.D."
+            surname="Mandyam">
+      <organization>Qualcomm Technologies Inc.</organization>
+      <address>
+        <postal>
+          <street>5775 Morehouse Drive</street>
+          <!-- Reorder these if your country does things differently -->
+          <city>San Diego</city>
+          <region>California</region>
+          <code>92121</code>
+          <country>USA</country>
+        </postal>
+        <phone>+1 858 651 7200</phone>
+        <email>mandyam@qti.qualcomm.com.com</email>
+        <!-- uri and facsimile elements may also be added -->
+      </address>
+    </author>
+
+    <date month="September" year="2016" />
+
     <keyword>webauthn</keyword>
     <keyword>attestation</keyword>
     <keyword>extensions</keyword>
     <keyword>registry</keyword>
-    
+
     <abstract>
-      <t>This specification defines IANA registries for W3C Web Authentication 
-      <xref target="WebAuthn"/>
-      attestation formats and extension identifiers.</t>
+      <t>
+        This specification defines IANA registries for W3C Web Authentication
+        <xref target="WebAuthn"/>
+        attestation formats and extension identifiers.
+      </t>
     </abstract>
-    
+
   </front>
-  
+
   <middle>
     <section title="Introduction">
       <t>
         This specification defines IANA registries for W3C Web Authentication <xref
-        target="WebAuthn"/> attestation formats and extension identifiers, and supplies initial 
+        target="WebAuthn"/> attestation formats and extension identifiers, and supplies initial
         entries within each registry.
       </t>
+      <t>
+        Candidate entries must be announced on the public Web Authentication mailing list,
+        public-webauthn@w3.org, so they can be discussed and evaluated for compliance 
+        to the requirements presented in this document before being added to its associated
+        registry.
+      </t>

```

Comment:
I do not agree with adding the above paragraph.  This is the sort of information that is addressed below where we say..

"The Expert(s) will establish procedures for requesting registrations, and make them available from the registry page. and make them available from the registry page."

..Mark Nottingham (mnot) indicated that their experience with RFC5988 and its rigid stipulation of registry rules was that it was too bureaucratic and inflexible (one has to create a new RFC to alter them (eg fine grained details like which mailing list, detail-level criteria, etc).  

Thus we should only say, in this spec, that the "experts" will make and publish the rules on the registry page.